src/main/kotlin/xyz/crossward/entities/IsMember.kt

```diff
@@ -1,14 +1,17 @@
 package xyz.crossward.entities
 
 import java.io.Serializable
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.Id
-import javax.persistence.Table
+import javax.persistence.*
 
 @Entity
 @Table(name = "IS_MEMBER")
+@IdClass(IsMemberId::class)
 data class IsMember(
     @Id @Column(name = "user_id") val userId: String,
     @Id @Column(name = "group_id") val groupId: String
+) : Serializable
+
+class IsMemberId(
```

Comment:
make this a `data class`. Can you also make the Wins ID class a `data class` as well?